src/sites/cclw/constants/griLinks.ts

```diff
@@ -0,0 +1,48 @@
+type TFooterItem = {
+  title: string;
+  links: TLinkItem[];
+};
+
+export type TLinkItem = {
+  text: string;
+  href: string;
+  external: boolean;
+};
+
+const navLinks: TFooterItem = {
+  title: "Grantham Research Institute",
+  links: [
+    {
+      text: "Grantham Research Institute",
+      href: "https://www.lse.ac.uk/granthaminstitute/",
+      external: true,
```

Comment:
Did we want to add `cy` attributes to these to for future testing?

Reply:
I currently have a test for GRI links that checks the length (for now)